react/src/components/tabs/components/SprkTabsButton/SprkTabsButton.js

```diff
@@ -20,18 +20,21 @@ class SprkTabsButton extends Component {
       tabBtnChildren,
       tabBtnId,
       tabBtnAddClasses,
+      tabBtnAdditionalClasses = tabBtnAddClasses,
       ariaControls,
       ariaSelected,
       isActive,
       onTabClick,
       tabBtnDataId,
       tabBtnAnalytics,
+      tabBtnAnalyticsString = tabBtnAnalytics,
     } = this.props;
     return (
       <button
         className={classnames(
           'sprk-c-Tabs__button',
-          tabBtnAddClasses,
+          tabBtnAdditionalClasses,
+          // eslint-disable-next-line prettier/prettier
           { 'sprk-c-Tabs__button--active': isActive },
         )}
```

Comment:
prettier formats it like this for me:
```suggestion
        className={classnames('sprk-c-Tabs__button', tabBtnAdditionalClasses, {
          'sprk-c-Tabs__button--active': isActive,
        })}
```

Reply:
My autoformatting turns it into something else but when I copy this suggestion in it works.